components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "defectdojo",
-  "version": "2.14.0-dev",
+  "version": "2.15.0-dev",
```

Comment:
why is it screaming about that in the diff, but the actual dev branch in defectdojo is on 2.15.0-dev ?!?!

Reply:
Unfortunately, I'm not able to help you here. I have no idea.
But I would try `git rebase dev`